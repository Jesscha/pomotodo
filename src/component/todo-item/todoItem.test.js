import React from 'react';
import { shallow } from 'enzyme';
import TodoItem from './todoItem.component';

TodoItem
describe("<TodoItem />", ()=>{
    const wrapper = shallow(<TodoItem />)
   


    it("renders TodoItem", ()=>{
        expect(wrapper).toMatchSnapshot();
    });

    

    
    })





