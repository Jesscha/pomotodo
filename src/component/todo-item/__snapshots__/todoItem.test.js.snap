// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoItem /> renders TodoItem 1`] = `ShallowWrapper {}`;
